n_request.c

```diff
@@ -750,4 +751,5 @@ NOTE_C_STATIC bool crcError(char *json, uint16_t shouldBeSeqno)
     uint32_t shouldBeCrc32 = crc32(json, fieldOffset);
     return (shouldBeSeqno != actualSeqno || shouldBeCrc32 != actualCrc32);
 }
-#endif
+
+#endif // !NOTE_LOWMEM
```

Comment:
There was an `#ifdef` guard on each of the CRC functions. I simplified it by just using one for all of them as a group.